hybridse/src/udf/default_udf_library.cc

```diff
@@ -2873,6 +2921,34 @@ void DefaultUdfLibrary::InitUdaf() {
         )")
         .args_in<int16_t, int32_t, int64_t, float, double>();
 
+    RegisterUdafTemplate<DrawdownUdafDef>("drawdown")
+        .doc(R"(
+            @brief Compute drawdown of values.
+
+            Drawdown is defined as the max decline percentage from a historical peak to a subsequent valley.
```

Comment:
so the base is the history peek value ?

Reply:
yes. the base is the history peak value.